Waralee  Documentation Update (#1813)
freesewing/freesewing: markdown/org/docs/patterns/waralee/cutting/en.md

<Note>

> Waralee pants parts consists of two mirrored pieces of fabric.

Reviewer: "Waralee pants parts consist" or "Waralee pants consist" or "Waralee consists"?

Author: Part when pattern, peice when fabric I believe.

They are basically big rectangles with pieces missing.

> As such, it is easier to print out the part you need to cut out and measure the rest,

Reviewer: I guess the idea is for this sentence to agree with the earlier one with regard to the usage of "piece" or "part".

Author: part for pattern

than it is to print out everything and cut the pattern the traditional way.

> However the pattern is available both ways so you can print out the whole thing, tape all the pages together, and use that to cut out the fabric if you prefer but we recommend only printing the part that you need to cut out.

Reviewer: This sentence felt a little long to me and I had to reread it. A small suggestion:
```suggestion
That said, the pattern is available both ways. So you can print out the whole thing, tape all the pages together, and use that to cut out the fabric, if you prefer. But we recommend only printing the part that you need to cut out.
```

Author: yea this better paced. I mostly just did basica reformat not wording work on this updated

> You will need to print out the pocket and pocket facings in the traditional way.

Reviewer: Maybe "pocket and pocket facing" or "pockets and pocket facings"?

Author: plural, typo

</Note>

### Materials
 - **Main fabric**

>    - Cut the **2 pants** parts.

Reviewer: Does this mean:
```suggestion
   - Cut **2 pants** parts.
```

or are there two parts for the pants?

Author: yea not sure why I added the "the" there

   - (Optional) Cut **2 Pocket** parts on the fold.
   - Cut **4 backPocket** parts
   - Cut **4 frontFacing** parts
   - Cut **4 backFacing** parts

<Note>

> You can cut the Pocket and backPocket out of a lining fabric if you prefer but this with only be needed if you deviate from the recommended main fabrics.

Reviewer: Small typo and a suggestion for making the number of main fabrics (1 I think) clearer:
```suggestion
You can cut the Pocket and backPocket out of a lining fabric if you prefer but this will only be needed if you deviate from the main fabric recommendations.
```

Author: I think just the will bit
